models/WikiPage.php

```diff
@@ -172,6 +176,8 @@ public function attributeLabels()
             'container_menu_order' => $isSpaceContainer
                 ? Yii::t('WikiModule.base', 'Sort order in Space menu')
                 : Yii::t('WikiModule.base', 'Sort order in Profile menu'),
+            'appendable_content' => 'Appendable Content',
+            'is_appendable' => 'Is Appendable',
```

Comment:
Please add a German translation, e.g. "Anhängbarer Inhalt" and "Ermögliche das Anhängen von Inhalt"